jazz_deployments-event-handler/test/test.js

```diff
@@ -0,0 +1,10 @@
+const assert = require('chai').assert;
+
+describe('Sample', function() {
+  it('tests handler', function(done) {
```

Comment:
I guess you will be working on these in this PR?

Reply:
No. Unit test has a separate story